proofs/cbmc/polyvec_basemul_acc_montgomery_cached/Makefile

```diff
@@ -13,7 +13,7 @@ DEFINES +=
 INCLUDES +=
 
 REMOVE_FUNCTION_BODY +=
-UNWINDSET += $(MLKEM_NAMESPACE)polyvec_basemul_acc_montgomery_cached.3:4 # Largest value of MLKEM_K
+UNWINDSET += $(MLKEM_NAMESPACE)polyvec_basemul_acc_montgomery_cached.0:4 # Largest value of MLKEM_K
```

Comment:
I forgot to comment on this in the commit message: The runtime assertions contain loops, hence the loop index changes with the switch to CBMC assertions.